aviary/variable_info/variable_meta_data.py

```diff
@@ -1749,6 +1724,20 @@
     default_value=0.0,
 )
 
+add_meta_data(
+    Aircraft.Engine.COMPUTE_PROPELLER_INSTALLATION_LOSS,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
```

Comment:
In GASP this is based on the value of INPROP.FT
if FT < 0, this bool is true, if >= 0, this is false and the value of FT is used as the installation loss factor